docs/example-scenario/apps/sap-dev-test.md

```diff
@@ -0,0 +1,109 @@
+
+### Alternatives
+
+* List of alternative options and why you might use them.
+
+## Considerations
+
+### Availability
+
+For any Single Instance Virtual Machine using premium storage for all Operating System Disks and Data Disks, we guarantee you will have Virtual Machine Connectivity of at least 99.9%.
+
+For more information on Microsoft Azure Service Level Agreement for Virtual Machines [SLA For Virtual Machines](https://azure.microsoft.com/en-us/support/legal/sla/virtual-machines/v1_8/)
+
+### Scalability
+
+For general guidance on designing scalable solutions, see the [scalability checklist][scalability] in the Azure Architecture Center.
+
+### Security
+
+For general guidance on designing secure solutions, see the [Azure Security Documentation][security].
+
+### Resiliency
+
+For general guidance on designing resilient solutions, see [Designing resilient applications for Azure][resiliency].
+
+## Pricing
+
+Explore the cost of running this scenario, all of the services are pre-configured in the cost calculator.  To see how the pricing would change for your particular use case change the appropriate variables to match your expected traffic.
+
+We have provided four sample cost profiles based on amount of traffic you expect to get:
+
+|Size|SAPs|VM Type|Storage|Azure Pricing Calculator|
+|----|----|-------|-------|---------------|
+|Small|8000|D8s_v3|2xP20, 1xP10|[Small](https://azure.com/e/9d26b9612da9466bb7a800eab56e71d1)|
+|Medium|16000|D16s_v3|3xP20, 1xP10|[Medium](https://azure.com/e/465bd07047d148baab032b2f461550cd)|
+Large|32000|E32s_v3|3xP20, 1xP10|[Large](https://azure.com/e/ada2e849d68b41c3839cc976000c6931)|
+Extra Large|64000|M64s|4xP20, 1xP10|[Extra Large](https://azure.com/e/975fb58a965c4fbbb54c5c9179c61cef)|
+
+Note: * pricing is a guide, indicates only the VMs and storage costs (excludes, networking, backup storage and data ingress/egress charges)
+
+* [Small](https://azure.com/e/9d26b9612da9466bb7a800eab56e71d1): A small system consists of VM type D8s_v3 with 8x vCPUs, 32GB RAM and 200GB temp storage, additionally two 512GB and one 128GB premium storage disks.
+* [Medium](https://azure.com/e/465bd07047d148baab032b2f461550cd): A medium system consists of VM type D16s_v3 with 16x vCPUs, 64GB RAM and 400GB temp storage, additionally three 512GB and one 128GB premium storage disks.
+* [Large](https://azure.com/e/ada2e849d68b41c3839cc976000c6931): A large system consists of VM type E32s_v3 with 32x vCPUs, 256GB RAM and 512GB temp storage, additionally three 512GB and one 128GB premium storage disks.
+* [Extra Large](https://azure.com/e/975fb58a965c4fbbb54c5c9179c61cef): An extra large system consists of a VM type M64s with 64x vCPUs, 1024GB RAM and 2000GB temp storage, additionally four 512GB and one 128GB premium storage disks.
+
+## Deployment
+
+To deploy a sample solution similar to the scenario above, please use the deploy button
```

Comment:
Should we call out that this doesn't deploy SAP on the VMs?

Reply:
Probably, I've updated this section to include a note that SAP does not get deployed just the infrastructure.